diff --git a/src/nusystematics/systproviders/GENIEReWeightEngineConfig.cc b/src/nusystematics/systproviders/GENIEReWeightEngineConfig.cc
index 4ca56cb..d6c70a2 100644
--- a/src/nusystematics/systproviders/GENIEReWeightEngineConfig.cc
+++ b/src/nusystematics/systproviders/GENIEReWeightEngineConfig.cc
@@ -300,7 +300,10 @@ ConfigureMECWeightEngine(SystMetaData const &MECmd,
         kXSecTwkDial_DecayAngMEC,
         kXSecTwkDial_FracPN_CCMEC,
         kXSecTwkDial_FracDelta_CCMEC,
-        kXSecTwkDial_XSecShape_CCMEC
+        kXSecTwkDial_XSecShape_CCMEC,
+        kXSecTwkDial_XSecShape_CCMEC_Empirical,
+        kXSecTwkDial_XSecShape_CCMEC_Martini,
+        kXSecTwkDial_EnergyDependence_CCMEC
       },
       "xsec_mec", []() { return new GReWeightXSecMEC; }, UseFullHERG, param_map);
 
