diff --git a/src/nusystematics/systproviders/GENIEReWeightEngineConfig.cc b/src/nusystematics/systproviders/GENIEReWeightEngineConfig.cc
index 4ca56cb..319f62c 100644
--- a/src/nusystematics/systproviders/GENIEReWeightEngineConfig.cc
+++ b/src/nusystematics/systproviders/GENIEReWeightEngineConfig.cc
@@ -298,9 +298,13 @@ ConfigureMECWeightEngine(SystMetaData const &MECmd,
         kXSecTwkDial_NormNCMEC,
         kXSecTwkDial_NormEMMEC,
         kXSecTwkDial_DecayAngMEC,
+        kXSecTwkDial_DecayAng2MEC,
         kXSecTwkDial_FracPN_CCMEC,
         kXSecTwkDial_FracDelta_CCMEC,
-        kXSecTwkDial_XSecShape_CCMEC
+        kXSecTwkDial_XSecShape_CCMEC,
+        kXSecTwkDial_XSecShape_CCMEC_Empirical,
+        kXSecTwkDial_XSecShape_CCMEC_Martini,
+        kXSecTwkDial_EnergyDependence_CCMEC
       },
       "xsec_mec", []() { return new GReWeightXSecMEC; }, UseFullHERG, param_map);
 
